com.github.rhyme_player.rhyme.yml

```diff
@@ -0,0 +1,66 @@
+app-id: com.github.rhyme_player.rhyme
+runtime: org.freedesktop.Platform
+runtime-version: "20.08"
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "20.08"
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node14
+command: run.sh
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
+  - --filesystem=home
```

Comment:
Consider limiting filesystem permissions to `xdg-music` (and probably adding removable media directories, i.e. `/media` and `/run/media`).

Reply:
The app can change the music directory to any where else on the os

Reply:
This will limit it only to the music folder

Reply:
> This will limit it only to the music folder

Yes, that's the point. Apps should try to stay secure-by-default, while full `home` access allows trivial sandbox escape. If your app doesn't modify the music directory, you can at least limit home access to read-only (changing this to `home:ro`).

Reply:
ok

Reply:
I will change it to read-only